django/website/contacts/models.py

```diff
@@ -110,3 +110,12 @@ def preferences(self):
 class UserPreferences(Model):
     user = OneToOneField(User, related_name='_preferences')
     last_viewed_logframe = ForeignKey('logframe.LogFrame', null=True)
+
+
+@python_2_unicode_compatible
+class NameOnlyPermission(Permission):
+    class Meta:
+        proxy = True
+
+    def __str__(self):
+        return "Can {0}".format(self.name.lower())
```

Comment:
Name only permissions are so called because only the name is displayed in it's string representation.